orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/utils/ClusterNameValidator.groovy

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Let's avoid new Groovy classes in `main` sourceset.

Reply:
thanks , updated to a java class.